.github/CODEOWNERS

```diff
@@ -143,14 +118,17 @@
 
 # Machine Learning
 /x-pack/plugins/ml/  @elastic/ml-ui
-/x-pack/test/functional/apps/machine_learning/  @elastic/ml-ui
-/x-pack/test/functional/services/machine_learning/  @elastic/ml-ui
-/x-pack/test/functional/services/ml.ts  @elastic/ml-ui
+/x-pack/test/functional/apps/ml/  @elastic/ml-ui
+/x-pack/test/functional/services/ml/  @elastic/ml-ui
 # ML team owns and maintains the transform plugin despite it living in the Elasticsearch management section.
 /x-pack/plugins/transform/  @elastic/ml-ui
 /x-pack/test/functional/apps/transform/ @elastic/ml-ui
-/x-pack/test/functional/services/transform_ui/ @elastic/ml-ui
-/x-pack/test/functional/services/transform.ts @elastic/ml-ui
+/x-pack/test/functional/services/transform/ @elastic/ml-ui/
+x-pack/test/api_integration_basic/apis/ml/ @elastic/ml-ui
```

Comment:
Shouldn't this have a leading `/`?

Reply:
Yes. Looks like the leading `/` accidentally went to the end of the previous line. Already fixed in #83233.

Reply:
🤦‍♀️